files/jquery.inputmask/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "jquery.inputmask",
+  "repo": "RobinHerbots/jquery.inputmask",
+  "files": {
+    "basePath": "dist/min/inputmask/",
+    "include": ["jquery.inputmask.min.js"]
```

Comment:
maybe, i don't know all syntax, i took example and edited. if somewhere i made a mistake, sorry :)